my-app/src/services/cookie.service.js

```diff
@@ -0,0 +1,17 @@
+import Cookies from "universal-cookie";
+import { Component } from "react";
+
+const cookies = new Cookies();
+export default class CookieService extends Component {
```

Comment:
Why do you use extends Component? Do you render something from this class?

Reply:
Should I use it only when I render smth?